README.md

```diff
@@ -1,9 +1,11 @@
 # jupyterhub-python-repo-template
 
-[![Documentation status](https://img.shields.io/readthedocs/jupyterhub-python-repo-template?logo=read-the-docs)](https://jupyterhub-python-repo-template.readthedocs.io/en/latest/?badge=latest)
-[![GitHub Workflow Status](https://img.shields.io/github/workflow/status/jupyterhub/jupyterhub-python-repo-template/Test?logo=github)](https://github.com/jupyterhub/jupyterhub-python-repo-template/actions)
-[![Code coverage](https://codecov.io/gh/jupyterhub/jupyterhub-python-repo-template/branch/main/graph/badge.svg)](https://codecov.io/gh/jupyterhub/jupyterhub-python-repo-template)
-[![](https://img.shields.io/pypi/v/jupyterhub-python-repo-template.svg?logo=pypi)](https://pypi.python.org/pypi/jupyterhub-python-repo-template)
+[![Latest PyPI version](https://img.shields.io/pypi/v/jupyterhub-python-repo-template?logo=pypi)](https://pypi.python.org/pypi/jupyterhub-python-repo-template)
+[![Latest conda-forge version](https://img.shields.io/conda/vn/conda-forge/jupyterhub-python-repo-template?logo=conda-forge)](https://anaconda.org/conda-forge/jupyterhub-python-repo-template)
+[![GitHub Workflow Status - Test](https://img.shields.io/github/actions/workflow/status/jupyterhub/jupyterhub-python-repo-template/test.yaml?logo=github&label=tests)](https://github.com/jupyterhub/jupyterhub-python-repo-template/actions)
+[![Test coverage of code](https://codecov.io/gh/jupyterhub/jupyterhub-python-repo-template/branch/main/graph/badge.svg)](https://codecov.io/gh/jupyterhub/jupyterhub-python-repo-template)
+[![GitHub](https://img.shields.io/badge/issue_tracking-github-blue?logo=github)](https://github.com/jupyterhub/jupyterhub-python-repo-template/issues)
+[![Discourse](https://img.shields.io/badge/help_forum-discourse-blue?logo=discourse)](https://discourse.jupyter.org/c/jupyterhub)
```

Comment:
Can we use a more generic word like "forum" or "discussion"? I don't think folks should need to know what Discourse is.

Reply:
The logo looks like this already:

![image](https://github.com/jupyterhub/jupyterhub-python-repo-template/assets/3837114/bd878602-4742-45b5-bb2c-0099a0cd66e0)

```suggestion
[![Help forum - Discourse](https://img.shields.io/badge/help_forum-discourse-blue?logo=discourse)](https://discourse.jupyter.org/c/jupyterhub)
```

Reply:
Matching the text in the badge is perfect!